CHANGELOG.md

```diff
@@ -43,6 +43,7 @@ Released on 2021-01-18
 
 ### Rule Changes
 
+* rule(Sudo Potential Privilege Escalation (CVE-2021-3156)): new rule created to detect CVE-2021-3156[[#1543](https://github.com/falcosecurity/falco/pull/1543)] - [@darryk5](https://github.com/darryk5)
```

Comment:
```suggestion
```

This patch will be not included in 0.27.0. Most important, do not modify the CHANGELOG.md manually, please. 
The changelog will be updated in the release process as described [here](https://github.com/falcosecurity/falco/blob/master/RELEASE.md#pre-release-checklist)